tests/tests/tier0/bluechi-change-port-with-c-cmd-option/test_bluechi_change_port_with_c_cmd_option.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+import os
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+from bluechi_test.util import read_file
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    node_foo = nodes[NODE_FOO]
+    config_file_location = "/var/tmp"
+    bluechi_agent_str = "bluechi-agent"
+    bluechi_controller_str = "bluechi-controller"
+    file_location_ctrl = os.path.join("config-files", "ctrl_port_8421.conf")
+    file_location_agent = os.path.join("config-files", "agent_port_8421.conf")
+
+    # Copying relevant config files into the nodes container
+    content = read_file(file_location_agent)
+    node_foo.create_file(config_file_location, file_location_agent, content)
+    content = read_file(file_location_ctrl)
+    ctrl.create_file(config_file_location, file_location_ctrl, content)
+
+    ctrl.restart_with_config_file(
+        os.path.join(config_file_location, "ctrl_port_8421.conf"), bluechi_controller_str)
+    assert ctrl.wait_for_unit_state_to_be(bluechi_controller_str, "active")
+
+    # Check if port 8421 is listenning and 8420 is disconnected
+    _, output = ctrl.exec_run("lsof -i:8421")
+    assert "bluechi-c" in str(output)
+    _, output = ctrl.exec_run("lsof -i:8420")
+    assert b'' == output
+
+    # Check if node disconnected
+    _, output = node_foo.exec_run("systemctl status bluechi-agent.service")
+    assert "Disconnected from manager" in output
```

Comment:
Please use the BlueChi API to check if a node is dis-/connected, e.g. like [here](https://github.com/eclipse-bluechi/bluechi/blob/main/tests/tests/tier0/monitor-node-reconnect/python/is_node_connected.py). 
Asserting on a log is rather brittle. For example, if we change the log level or just one character this test is breaking.

Reply:
Chnaged to use BlueChi API